doc/source/user/plots/matplotlib1.py

```diff
@@ -3,5 +3,5 @@
 
 a = np.array([2, 1, 5, 7, 4, 6, 8, 14, 10, 9, 18, 20, 22])
 
-plt.plot(a) 
-plt.show()
\ No newline at end of file
+plt.plot(a)
+plt.show()
```

Comment:
Are these removing newlines? The Unix standard is that text files end in newlines.

Reply:
No, they add the missing newline at EOF.